57.md

```diff
@@ -96,7 +96,11 @@ Example zap note:
       [
         "preimage",
         "5d006d2cf1e73c7148e7519a4c68adc81642ce0e25a432b2434c99f97344c15f"
-      ]
+      ],
```

Comment:
This tag should be inside the 9734 not the 9735

Reply:
Fixed